docs/patterns/components/Modal/index.js

```diff
@@ -72,7 +72,7 @@ export default class Demo extends Component {
           </ModalFooter>
         </Modal>
         <h2>Code Sample</h2>
-        <Code langauge="javascript" role="region" tabIndex={0}>
+        <Code language="javascript" role="region" tabIndex={0}>
```

Comment:
It doesn't look any different with the typo fix. 
<img width="961" alt="Screen Shot 2022-09-15 at 2 18 05 PM" src="https://user-images.githubusercontent.com/3081483/190500731-93607ca7-56c2-4a2d-9c24-e5009d7b2ae0.png">